tests/test_checkpoint.rs

```diff
@@ -0,0 +1,121 @@
+// Copyright 2018 Eugene P.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+extern crate rocksdb;
+
+use rocksdb::{checkpoint::Checkpoint, DB, Options};
+
+fn remove_dir_silently(path: &str) {
+    use std::fs::remove_dir_all;
+
+    match remove_dir_all(&path) {
```

Comment:
If you want to ignore result, I think `let _ = remove_dir_all(&path)` is more readable.